From 186ec0c80410b2120b8f31c211cb44a836c84522 Mon Sep 17 00:00:00 2001
From: Haihua Hu <jared.hu@nxp.com>
Date: Wed, 4 Jul 2018 16:05:20 +0800
Subject: [PATCH] weston.ini: using argb8888 as gbm default on mscale 850D

mscale 850D video playback need using pixel alpha for graphic layer

upstream status: imx specific
Signed-off-by: Haihua Hu <jared.hu@nxp.com>

---
 weston.ini.in | 1 +
 1 file changed, 1 insertion(+)

Index: git/weston.ini.in
===================================================================
--- git.orig/weston.ini.in
+++ git/weston.ini.in
@@ -2,6 +2,7 @@
 # i.MX: Disable idle timeout
 idle-time=0
 #use-g2d=1
+gbm-format=argb8888
 
 [libinput]
 touchscreen_calibrator=true
